### PR TITLE
Update color-tokens.js

### PR DESCRIPTION
**Pull request by lee-chase**
Does not match current Carbon source code for overlay color.

The current Carbon source code uses different colors for overlay. This does not add AI overlay.

image

### CODE REVIEW

File: src/data/guidelines/color-tokens.js [hunk 1]
```diff
@@ -2288,20 +2288,20 @@ const colorTokens = {
...
-          name: 'Gray 100, 50%',
-          hex: '#161616 @ 50%',
+          name: '#222222, 50%',
+          hex: '#222222 @ 50%',
         },
         g90: {
-          name: 'Gray 100, 70%',
-          hex: '#161616 @70%',
+          name: 'Black, 65%',
+          hex: '#000000 @65%',
```
Comment (alisonjoseph) [inline src/data/guidelines/color-tokens.js:2300]:
```suggestion
          hex: '#000000 @ 65%',
```

**Review by alisonjoseph — APPROVED** [review]
LGTM, just minor missing space

**Review by guidari — APPROVED** [review]

**Comment by laurenmrice** [comment]
Hey @lee-chase ! 

The [color values](https://carbondesignsystem.com/elements/color/tokens) on the website for `$overlay` are the correct specs. We actually need to be updating the code with what the website currently states.

---

`$overlay`

| White theme | Gray 10 theme | Gray 90 theme | Gray 100 theme |
|--------|--------|--------| --- |
| Gray 100 at 50% | Gray 100 at 50% | Gray 100 at 70% | Gray 100 at 70% |

**Comment by alisonjoseph** [comment]
Going to close this since we need to update the code not the website. 😅

**Pull request closed without merging**